stdlib/private/OSLog/StringAlignment.swift

```diff
@@ -0,0 +1,67 @@
+@frozen
+public enum CollectionBound {
```

Comment:
I'd pick a specific name to avoid potential future conflicts or shadowing like OSCollectionBound